vms/evm/uptimetracker/state.go

```diff
@@ -138,33 +132,20 @@ func (s *state) GetStartTime(nodeID ids.NodeID) (time.Time, error) {
 
 func (s *state) addValidatorUpdate(vdr *validator) {
 	s.addValidator(vdr.validationID, vdr)
-	s.updates[vdr.validationID] = updatedStatus
+	s.updatedValidators.Add(vdr.validationID)
 }
 
-func (s *state) updateValidator(
-	validationID ids.ID,
-	isActive bool,
-	weight uint64,
-) error {
-	v, ok := s.validators[validationID]
-	if !ok {
-		return database.ErrNotFound
-	}
```

Comment:
This not possible to hit - we always do a check that the validator was present before calling this function